solr/managed-schema.xml

```diff
@@ -63,6 +74,7 @@
   <field name="experiment_id" type="extra_facet" stored="true" indexed="true" multiValued="true" />
   <field name="member_id" type="extra_facet" stored="true" indexed="true" multiValued="true" />
   <field name="table_id" type="extra_facet" stored="true" indexed="true" multiValued="true" />
+  <field name="frequency" type="extra_facet" stored="true" indexed="true" multiValued="true"/>
```

Comment:
is this the one that will gather 

```
time_reduction: {'inst', 'inst-or-acc', 'unknown', 'mean'}
```
type of information?

Reply:
I am unable to directly comment on them but these are not used in the schema, should we drop them? shoudl we jkeep them?

```xml
  <fieldType name="pdates" class="solr.DatePointField" docValues="true" multiValued="true"/>
  <fieldType name="pdoubles" class="solr.DoublePointField" docValues="true" multiValued="true"/>
  <fieldType name="plongs" class="solr.LongPointField" docValues="true" multiValued="true"/>
  <fieldType name="booleans" class="solr.BoolField" sortMissingLast="true" multiValued="true"/>
```

Reply:
> is this the one that will gather
> 
> ```
> time_reduction: {'inst', 'inst-or-acc', 'unknown', 'mean'}
> ```
> 
> type of information?

No, that's time_frequncy. If time_reduction is not present, then I've forgotten it.

Reply:
> I am unable to directly comment on them but these are not used in the schema, should we drop them? shoudl we jkeep them?
> 
> ```
>   <fieldType name="pdates" class="solr.DatePointField" docValues="true" multiValued="true"/>
>   <fieldType name="pdoubles" class="solr.DoublePointField" docValues="true" multiValued="true"/>
>   <fieldType name="plongs" class="solr.LongPointField" docValues="true" multiValued="true"/>
>   <fieldType name="booleans" class="solr.BoolField" sortMissingLast="true" multiValued="true"/>
> ```

I don't know, this is "that has been there before I cam TM".

Reply:
I left them there because Solr didn't want to start the core without them. Keep them in the file.

Reply:
My bad, `time_reduction` is also present. I see that `frequency` is among the facet you mentioned in the rocketchat


then LGTM

Reply:
I have one question: when we do a query can we search through this "hidden (?)" extra_facets? E.g.

```shell
freva databrowser mip_era="foo" source_id="bar" ...
``

Reply:
Yes, that is the idea.